### PR TITLE
Add support for Jekyll 4.x

### PR DESCRIPTION
**Pull request by dhoppe**

### CODE REVIEW

File: jekyll-maps.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r!^(test|spec|features)/!) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "jekyll", "~> 3.0"
+  soec.add_dependency "jekyll", ">= 3.0", "< 5.0"
```
Comment (ayastreb) [inline jekyll-maps.gemspec]: Should it be `spec`?

**Comment by dhoppe** [comment]
@ayastre it only changes the requirements to make this plugin compatible with the latest release of Jekyll. For example:
https://github.com/benbalter/jekyll-relative-links/pull/57/files

File: jekyll-maps.gemspec (hunk 1 above)
Comment (dhoppe) [reply jekyll-maps.gemspec]: @ayastreb I just fixed it.

**Comment by rriemann** [comment]
Looking forward to see this merged!

**Comment by ayastreb** [comment]
Merged and released in v2.3.2

**Pull request merged**